stack-build

```diff
@@ -19,7 +19,7 @@ get_build_id () {
 }
 
 usage () {
-    echo "usage: stack-build -n name -r recipe-path -s system -b build-root"
```

Comment:
How do you intend to enable this flag on a per-recipe basis?

This script is called here:

https://github.com/eth-cscs/alps-spack-stacks/blob/main/ci/common.yml

And this flag would need to be enabled on a per recipe/target basis, i.e. probably by setting a variable here:

https://github.com/eth-cscs/alps-spack-stacks/blob/main/recipes/mch/ci.yml

Reply:
Good question. I did not add such functionality here because it would have been untested. I already added a `SPACK_DEVELOP` variable in #36 (untested) which is now a string but I'd like to change to a boolean (defaulting to `false` to match the current behavior).

Reply:
Tested in #36.